components/content_settings/renderer/brave_content_settings_agent_impl.cc

```diff
@@ -229,6 +229,49 @@ bool BraveContentSettingsAgentImpl::AllowFingerprinting(
   return GetBraveFarblingLevel() != BraveFarblingLevel::MAXIMUM;
 }
 
+bool BraveContentSettingsAgentImpl::IsCosmeticFilteringEnabled(
```

Comment:
tests for these methods?

Reply:
These are already tested through `Default1pBlockingFlagDisabledTest.Aggressive1pBlocking`, `CosmeticFilteringFlagDisabledTest.*`, and `AdBlockServiceTest.CosmeticFiltering*`; what else do you want to see added?

Reply:
we should have unit tests for these methods